Sources/EventLogger.swift

```diff
@@ -14,15 +14,15 @@ public enum LoggingEvent {
 		case value, completed, failed, terminated, disposed, interrupted
 
 		public static let allEvents: Set<Signal> = [
-			.value, .completed, .failed, .terminated, .disposed, .interrupted,
+			.value, .completed, .failed, .terminated, .disposed, .interrupted
```

Comment:
Can we change this rule? By default SwiftLint enforces no trailing comma. I think we should enforce the trailing comma. This tends to make diffs more readable and merges easier since adding a line doesn't alter the previously last item in the array.